Common/yaml/admin.yaml

```diff
@@ -0,0 +1,319 @@
+   
+  
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: nk5469562@gmail.com
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.11
+externalDocs:
+  description: Find out more about Swagger
+  url: http://swagger.io
+servers:
+  - url: http://localhost:8080
+tags:
+  - name: contacts
+    description: Apis relatd to contacts
+    externalDocs:
+      description: Find out more
+      url: http://swagger.io
+  
+paths:
+  /v1/contacts/admin/users/{userId}:
+    get:
+      tags:
+        - contacts
+      summary: Find contacts 
+      description: To get list of contatcs of specific user 
+      operationId: getListOFContact
+      
+      parameters:
+        - name: userId
+          in: path
+          description: userId values that need to be considered for filter
+          required: true
+          schema:
+            type: integer
+            example: 1001
+            
+        - name: contactIds
+          in: query
+          description: contactId values that need to be considered for filter
+          required: false
+          schema:
+            type: string
+            default: '101,102,103'
+            enum:
+              - '101'
+              - '101,102,103'
+              
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/contactList'
+                
+                example:
+                  - id: 101
+                    fullName: Naneet Kumar
+                    contacatNumber: 7546930695
+                    personalEmail: nk5469562@gmail.com
+                    
+                  - id: 102
+                    fullName: Mukesh Kumar
+                    contacatNumber: 7546930799
+                    personalEmail: mukesh@gmail.com  
+                  
+            
+        '400':
+          description: BAD REQUEST 
+          content:
+             application/json:
+              schema:
+                type: string
+                example: 
+                  message: Invalid UserId/ContactId
+                  
+          
+        '404':
+          description: NOT FOUND  
+          content:
+             application/json:
+              schema:
+                type: string
+                example: 
+                  message: Contact not found
+                  
+        '500':
+          description: INTERNAL SERVER ERROR
+          content:
+             application/json:
+              schema:
+                type: string
+                example: 
+                  message: Internal Server Error
+                  
+                
+    delete:
+      tags:
+           - contacts
```

Comment:
tags should always start with uppercase character

Reply:
fixed